pkg/rds/tags.go

```diff
@@ -0,0 +1,49 @@
+package rds
+
+import (
+	"errors"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/rds"
+)
+
+// DetermineArn returns the ARN for an RDS instance or cluster given the database name
+// It could return 2 ARNs if a cluster and instance with the same name exist
+func (cl Client) DetermineArn(db *string) ([]string, error) {
+	arns := []string{}
+
+	log.Println("Trying to determine ARN for", *db)
```

Comment:
it makes sense to stick to `aws.String[Value]` imo even though its longer (or not use the helpers at all).